eyieko/landville-frontend
# #165826890 User should search for properties

## Description ##
- A user needs to be able to search for properties

## Type of change ##
Please select the relevant option

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes

## How This Can Be Manually Tested? ##
- Check out the review app [here](https://landville-frontend-pr-28.herokuapp.com/)
- Pass is search queries into the search fields and search.

> OR

- Clone the repo [here](https://github.com/landvilleng/landville-frontend.git)
- Checkout to `ft-search-165826890`
- Run `npm i` to install dependencies
- Run `ng serve -o` to run the application on your default browser
- Pass is search queries into the search fields and search.

## Checklist: ##

## Related Pivotal Tracker stories ##
[#165826890](https://www.pivotaltracker.com/story/show/165826890)

[Finishes #165826890]
